### PR TITLE
Added guide for Cryptpad

### PR DESCRIPTION
**Pull request by hum6ug**

### CODE REVIEW

File: source/guide_cryptpad.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  [isabell@stardust html]$ cp config.example.js config.js
+
+Change the value of the variable ``_domain`` to match your newly created ``https://isabell.uber.space``.
+
+Due to the Web Backends we don't need to mess with ``.htaccess`` or any other configuration.
+
+Setup daemon
+------------
+
+I had some problems using ``supervisorctl`` so I changed it to the recommended ``forever.js``.
```
Comment (luto) [inline source/guide_cryptpad.rst]: Which problems did you have with supervisord? I am sure we can work through them!

Comment (hum6ug) [reply source/guide_cryptpad.rst]: Supervisord kept restarting. It looked like supervisord did not realized it was already running fine.

Comment (hum6ug) [reply source/guide_cryptpad.rst]: Okay now it is working with supervisorctl. I just had a problem with the working directory

**Comment by luto** [comment]
I think you are the first guide to use web backends out of the box :tada: I'll take a closer look, once we've merged the instructions in the manual, which should happen soon :)

**Comment by hum6ug** [comment]
Well, Okay :D

File: source/guide_cryptpad.rst [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+::
+
+  [isabell@stardust ~]$ uberspace web backend set isabell.uber.space --http --port 3000
+  Set backend for isabell.uber.space/ to port 3000; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+Now let's get started with Cryptpad.
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 6:
```
Comment (SalocinHB) [inline source/guide_cryptpad.rst]: Node 6 will reach its EOL next month, which means we will remove it from U7 hosts. I just built Cryptpad with Node 10 and it seems to work just fine …

File: source/guide_cryptpad.rst [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Start with cloning the Cryptpad source code from Github_ and be sure to replace the branch ``2.18.0`` with the current release number from the feed_:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone --branch 2.18.0 https://github.com/xwiki-labs/cryptpad.git ~/html
```
Comment (SalocinHB) [inline source/guide_cryptpad.rst]: node scripts do *not* belong in the Document Root. Use `~/cryptpad` instead

File: source/guide_cryptpad.rst [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+ command=node server
+ autorestart=yes
+
+Now let's start the service:
+
+.. code-block:: console
+
+ [isabell@stardust html]$ supervisorctl reread
+ [isabell@stardust html]$ supervisorctl update
+ [isabell@stardust html]$ supervisorctl start cryptpad
```
Comment (SalocinHB) [inline source/guide_cryptpad.rst]: `supervisorctl update` will attempt to start any new services, no need to do it manually

File: source/guide_cryptpad.rst [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+After we're enrolled into the beta, we need to setup the Web Backends:
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set isabell.uber.space --http --port 3000
```
[the remark this answers is not in the document]
Comment (hum6ug) [reply source/guide_cryptpad.rst]: Okay sounds good

File: source/guide_cryptpad.rst (hunk 2 above)
Comment (hum6ug) [reply source/guide_cryptpad.rst]: Alright

File: source/guide_cryptpad.rst (hunk 3 above)
Comment (hum6ug) [reply source/guide_cryptpad.rst]: Okay good too know

File: source/guide_cryptpad.rst (hunk 4 above)
Comment (hum6ug) [reply source/guide_cryptpad.rst]: Great

**Comment by hum6ug** [comment]
Sorry. Git config got messed up. That's why there is my old username in the commit

**Comment by hum6ug** [comment]
Do I have to do something to get rid of "Changes requested"? I think I changed everything. Or do we have to wait until Web Backends are in the official manual of U7?

**Pull request merged**